PluginGenerator/RulesPluginBuilder.cs

```diff
@@ -88,7 +96,7 @@ protected override void ApplyConfiguration(string baseWorkingDirectory)
             this.AddRuleSources(tempDir);
             this.AddRuleJars();
 
-            this.SetSourceCodeTokenReplacement(WellKnownSourceCodeTokens.Rule_PluginId, this.FindPluginKey());
```

Comment:
_Rule_PluginId_ was a misleading name: it's actually the unique part of the repository id.